src/reducers/workspaces.ts

```diff
@@ -229,28 +263,21 @@ export const reducer: Reducer<IWorkspaceManagerState> = (
     case SEND_REPL_INPUT_TO_OUTPUT:
       // CodeOutput properties exist in parallel with workspaceLocation
       newOutput = state[location].output.concat(action.payload as CodeOutput)
-      let newReplHistoryRecords: string[]
-      if (action.payload.value !== '') {
-        newReplHistoryRecords = [action.payload.value].concat(state[location].replHistory.records)
-      } else {
-        newReplHistoryRecords = state[location].replHistory.records
-      }
-      if (newReplHistoryRecords.length > maxBrowseIndex) {
-        newReplHistoryRecords.pop()
-      }
       return {
         ...state,
         [location]: {
           ...state[location],
-          output: newOutput,
-          replHistory: {
-            ...state[location].replHistory,
-            records: newReplHistoryRecords
-          }
+          output: newOutput
         }
       }
     case EVAL_EDITOR:
-      // Forces re-render of workspace on editor eval
+      return {
```

Comment:
Can we preserve the comment `// Forces re-render of...`? I'm 100% gonna forget what this does in the future and end up deleting it again.

Reply:
My bad, got removed in a rebase. I intended to keep it too